src/main/java/org/wildfly/security/util/Alphabet.java

```diff
@@ -269,4 +269,41 @@ public int decode(final int codePoint) {
             }
         };
     }
+
+    /**
+     * The alphabet used by PicketBox project base 64 encoding.
+     * {@code 0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz./}
+     */
+    public static final Base64Alphabet PICKETBOX_BASE_64 = new Base64Alphabet(false) {
```

Comment:
It is not standard alphabet which is used in PicketBox. I will strip the _BASE_64 suffix, but alphabet is needed for compatibility with older masked passwords.